### PR TITLE
Fixing workers not repairing tiles with Great improvement when pillaged

### PR DESCRIPTION
**Pull request by Emandac**
With this code the worker ai should fix pillaged tiles with Great improvement.

### CODE REVIEW

**Comment by EmperorPinguin** [comment]
You need to clean up this other stuff you used.

**Comment by Emandac** [comment]
> a couple of added/d



> You need to clean up this other stuff you used for the city settling issue, also a couple of added/deleted spaces (you can look at "Files changed" above)

Done 👍

File: core/src/com/unciv/logic/automation/unit/SpecificUnitAutomation.kt [hunk 1]
```diff
@@ -190,7 +190,6 @@ object SpecificUnitAutomation {
                 throw Exception("City within distance")
             return
         }
-
```
Comment (yairm210) [inline core/src/com/unciv/logic/automation/unit/SpecificUnitAutomation.kt:193]: Undo

File: core/src/com/unciv/logic/automation/unit/WorkerAutomation.kt [hunk 2]
```diff
@@ -270,8 +270,6 @@ class WorkerAutomation(
         tileRankings[tile] = TileImprovementRank(priority)
         return priority + unitSpecificPriority
     }
-
```
Comment (yairm210) [inline core/src/com/unciv/logic/automation/unit/WorkerAutomation.kt:273]: Undo

**Review by yairm210 — APPROVED** [review]

**Pull request merged**